File: Medii de proiectare si programare/C#/tema7/networking/LogOutRequest.cs
using System;
using model.domains;

namespace networking
{
    [Serializable]
    public class LogOutRequest : Request
    {
        public LogOutRequest(EmployeeAtOffice employeeAtOffice)
        {
            EmployeeAtOffice = employeeAtOffice;
        }

        public EmployeeAtOffice EmployeeAtOffice { get; }
    }
}

File: an2/Medii de proiectare si programare/java/tema7/temaJavaMPP/ChatNetworking/src/main/java/com/example/objectProtocol/LogOutRequest.java
package com.example.objectProtocol;

import com.example.EmployeeAtOffice;

public class LogOutRequest implements Request {
    private EmployeeAtOffice employeeAtOffice;

    public LogOutRequest(EmployeeAtOffice employeeAtOffice) {
        this.employeeAtOffice = employeeAtOffice;
    }

    public EmployeeAtOffice getEmployeeAtOffice() {
        return employeeAtOffice;
    }
}
